src/components/TopBar/TopBar.jsx

```diff
@@ -7,6 +7,42 @@ import ThemeSwitcher from "../ThemeSwitch/ThemeSwitch";
 import { Flex } from "rimble-ui";
 import useMediaQuery from "@material-ui/core/useMediaQuery";
 import "./topbar.scss";
+import { ReactComponent as OhmIcon } from '../../assets/icons/ohm-mini-logo.svg';
+import { ReactComponent as SohmIcon } from '../../assets/icons/sohm-gold-logo.svg';
+import ToggleButton from "@material-ui/lab/ToggleButton";
+
+const SOHM_ADDRESS = '0x04f2694c8fcee23e8fd0dfea1d4f5bb8c352111f';
+const OHM_ADDRESS = '0x383518188c0c6d7730d91b2c03a03c837814a899';
+const TOKEN_DECIMALS = 9;
+const TOKEN_IMAGE = 'https://app.olympusdao.finance/static/media/logo.b949cd68.svg';
+
+
+const addOhmToWallet = (tokenSymbol, tokenAddress) => async () => {
+  try {
+    const wasAdded = await window.ethereum.request({
```

Comment:
So `window.ethereum` only works for metamask. We shouldn't always assume its going to be there either. Is there any other way to do this or is this a Metamask only feature?

Reply:
Fixed.